src/core/blockchain/types/config.ts

```diff
@@ -4,10 +4,23 @@ import { ChainIdType } from '.';
 import { ITokenConfigState } from '../../../redux/tokens/state';
 import { IAffiliateBannerType } from '../../../components/affiliate-banner/types';
 
+export enum BlockchainNameService {
```

Comment:
these addresses are the same for testnets too ?

Reply:
On the ziliqa blockchain: only testnet is currently supported. 
On the eth: the address works with mainnet and testnet aswel